### DISCUSSION TITLE
Run docker image GPU error on branch master

### DISCUSSION THREAD

**Discussion by pisken** [OvenMediaLabs/OvenMediaEngine#1436] [Help]
**Describe the bug**
I just get branch master and build image for gpu, and build successfully, but when run show error 'The 'HardwareAcceleration' option is deprecated'. Although, I don't config HardwareAcceleration and only config for 'HWAccels', when I config multiples application using include in xml

**To Reproduce**

**Expected behavior**
allow check rule HWAccels in include applications

**Logs**
<img width="1440" alt="image" src="https://github.com/AirenSoft/OvenMediaEngine/assets/8767534/3bf8ff28-c472-4f32-96f4-6ad825ba4cca">

**Server (please complete the following information):**
 - Branch: master

**Additional context**
This is config in Server.xml
<img width="574" alt="image" src="https://github.com/AirenSoft/OvenMediaEngine/assets/8767534/a498d637-1a77-4ffe-b67b-aec60fc974a1">

This is config for app/demo.xml
<img width="607" alt="image" src="https://github.com/AirenSoft/OvenMediaEngine/assets/8767534/bc1abc95-0f87-4cad-b47b-0f3a68326379">

**Comment by pisken** [comment]
I was checked, certainly not setting 'HardwareAcceleration'. I remove all sub apps, there's only demo app.

**Reply by Keukhan** [reply]
Even if the HardwareAcceleration option exists, the program has been improved so that it is not forcibly terminated and only a warning message is displayed. Please try again with the latest version.

**Reply by pisken** [reply]
thanks, I will retry it
